content/docs/guides/hedgedoc.md

```diff
@@ -0,0 +1,144 @@
+# HedgeDoc
+
+ Use Pomerium to secure access to HedgeDoc by providing identity and context. 
+
+ ## What is HedgeDoc?
+
+[HedgeDoc](https://hedgedoc.org/) is a collaborative, web-based Markdown editor that allows you to create notes, graphs, and diagrams in your browser. 
+
+You can share a link to your document with other members of your organization so you can collaborate in real time.
+
+ ## Integrate Pomerium with HedgeDoc
+ 
+ HedgeDoc is open-source, self-hosted software that allows you to configure authentication and authorization with a [reverse proxy](https://docs.hedgedoc.org/guides/reverse-proxy/). 
+ 
+ You can configure Pomerium so that it authenticates and authorizes users against an Identity Provider (IdP) before users can access HedgeDoc's services.
+
+The flow looks like this: 
+
+1. User requests to access the secured app
+2. Pomerium proxy identifies user
+3. Pomerium proxy checks if user is authorized to access routes defined in the authorization [`policy`](https://www.pomerium.com/docs/reference/routes/policy)
+4. If authorized, Pomerium proxy authenticates user against IdP
+5. After IdP establishes user's identity, Pomerium proxy checks permissions against the secured app
+6. User can make authorized requests in the secured app
+
+![Pomerium IAP flow](../img/pom-auth-flow.png)
+
+### Set up your environment
```

Comment:
Thoughts on linking to the docker-based quick start instead, and just referencing that we assume users have a completed version of the quick-start (docker  based) as a starting point?